### ISSUE TITLE
Error importing material modules

### ISSUE THREAD

**Issue by theIDinside** [luixaviles/socket-io-typescript-chat#8]
When trying to build the client side, one gets a lot of errors, regarding the importing of material modules.
Currently, they're for example referenced by 
```javascript
import {MdButtonModule} from '@angular/material'
```

When following the build instructions, one gets errors like this, when trying to 'ng serve':
```bash
ERROR in Error: MdButtonModule is not an NgModule
    at _getNgModuleMetadata (/home/cx/workspace/es/socket-io-typescript-chat/client/node_modules/@angular/compiler-cli/src/ngtools_impl.js:140:15)
    at _extractLazyRoutesFromStaticModule (/home/cx/workspace/es/socket-io-typescript-chat/client/node_modules/@angular/compiler-cli/src/ngtools_impl.js:109:26)
    at /home/cx/workspace/es/socket-io-typescript-chat/client/node_modules/@angular/compiler-cli/src/ngtools_impl.js:129:27
    at Array.reduce (<anonymous>)
    at _extractLazyRoutesFromStaticModule (/home/cx/workspace/es/socket-io-typescript-chat/client/node_modules/@angular/compiler-cli/src/ngtools_impl.js:128:10)
    at Object.listLazyRoutesOfModule (/home/cx/workspace/es/socket-io-typescript-chat/client/node_modules/@angular/compiler-cli/src/ngtools_impl.js:53:22)
    at Function.NgTools_InternalApi_NG_2.listLazyRoutes (/home/cx/workspace/es/socket-io-typescript-chat/client/node_modules/@angular/compiler-cli/src/ngtools_api.js:91:39)
    at AotPlugin._getLazyRoutesFromNgtools (/home/cx/workspace/es/socket-io-typescript-chat/client/node_modules/@ngtools/webpack/src/plugin.js:241:66)
    at _donePromise.Promise.resolve.then.then.then.then.then (/home/cx/workspace/es/socket-io-typescript-chat/client/node_modules/@ngtools/webpack/src/plugin.js:495:24)
    at <anonymous>
    at process._tickCallback (internal/process/next_tick.js:188:7)
ERROR in /home/cx/workspace/es/socket-io-typescript-chat/client/src/app/dialog-user/dialog-user.component.ts (2,23): Module '"/home/cx/workspace/es/socket-io-typescript-chat/client/node_modules/@angular/material/material"' has no exported member 'MD_DIALOG_DATA'.
```

**Comment by theIDinside** [comment]
PR #9 sets out to resolve this issue.

**Comment by luixaviles** [comment]
Closed with #10
